.gitignore

```diff
@@ -106,3 +106,6 @@ venv.bak/
 # deployment package
 package.zip
 package/
+
+# vscode
+.vscode
```

Comment:
I think it's preferred to list IDE-specific ignores to your global git ignore instead.

Reply:
I also use vscode but typically put the ignores into local ``.git/info/exclude``, I don't mind having this kind of thing added to the repo though.

Reply:
I have moved it to the global git ignore.